scalardb-sample/docker-compose.yml

```diff
@@ -1,5 +1,25 @@
 version: "3.5"
 services:
+  mysql:
```

Comment:
@brfrn169 @Torch3333
Are we adding the Oracle database later?
For Cosmos DB, can we use it?
https://github.com/Azure/azure-cosmos-db-emulator-docker

Reply:
@feeblefakie 

> Are we adding the Oracle database later?

We can do that. However, since Oracle doesn't provide container images for Oracle databases, users need to build the images by following https://github.com/oracle/docker-images. 

> For Cosmos DB, can we use it?

Yes, I think so. However, as we know, the Cosmos DB container image for Linux has some limitations, so it might take some time to investigate and verify it. Alternatively, we might be able to provide it only for Windows users.

@Torch3333 What do you think?

Reply:
**Regarding Oracle,** 
I found out recently that they do provide containers on their [docker registry. ](https://container-registry.oracle.com/)

There is a free image (2.4GB) for [Oracle DB 23 ](https://container-registry.oracle.com/ords/ocr/ba/database/free ) that does not require any configuration to pull but it takes several minutes for the container to be ready to use
```
  oracle:
    image: "container-registry.oracle.com/database/free:latest"
    environment:
      ORACLE_PWD: Oracle
    ports:
      - "1521:1521"
```
I tried using the lightweight (870MB) version `23.4.0.0-lite` but ScalarDB fails to create a namespace so we can't use it now.

**Regarding Cosmos**
Besides what is mentioned, we also need to import Cosmos TLS/SSL certificate into the JDK cert store which is a bit troublesome, cf. how we do it on the [CI](https://github.com/scalar-labs/scalardb/blob/0aa73279582a49ddf68cc8a69741f2fdfcff338c/.github/workflows/ci.yaml#L356-L375).